[2022-11-26T00:33:49.231+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: cp_webcheck_failed_copy.http_check_copy scheduled__2022-11-25T00:00:00+00:00 [queued]>
[2022-11-26T00:33:49.249+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: cp_webcheck_failed_copy.http_check_copy scheduled__2022-11-25T00:00:00+00:00 [queued]>
[2022-11-26T00:33:49.250+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-26T00:33:49.252+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-26T00:33:49.254+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-26T00:33:49.295+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): http_check_copy> on 2022-11-25 00:00:00+00:00
[2022-11-26T00:33:49.319+0000] {standard_task_runner.py:55} INFO - Started process 6042 to run task
[2022-11-26T00:33:49.331+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'cp_webcheck_failed_copy', 'http_check_copy', 'scheduled__2022-11-25T00:00:00+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/cp_webcheck_failed copy.py', '--cfg-path', '/tmp/tmpxlxugne4']
[2022-11-26T00:33:49.334+0000] {standard_task_runner.py:83} INFO - Job 90: Subtask http_check_copy
[2022-11-26T00:33:49.503+0000] {task_command.py:376} INFO - Running <TaskInstance: cp_webcheck_failed_copy.http_check_copy scheduled__2022-11-25T00:00:00+00:00 [running]> on host 71a129efd3d8
[2022-11-26T00:33:49.671+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cp_webcheck_failed_copy
AIRFLOW_CTX_TASK_ID=http_check_copy
AIRFLOW_CTX_EXECUTION_DATE=2022-11-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-25T00:00:00+00:00
[2022-11-26T00:33:49.676+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-11-26T00:33:49.682+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'curl -s https://localhost > /dev/null']
[2022-11-26T00:33:49.720+0000] {subprocess.py:86} INFO - Output:
[2022-11-26T00:33:49.768+0000] {subprocess.py:97} INFO - Command exited with return code 7
[2022-11-26T00:33:49.789+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 7.
[2022-11-26T00:33:49.801+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=cp_webcheck_failed_copy, task_id=http_check_copy, execution_date=20221125T000000, start_date=20221126T003349, end_date=20221126T003349
[2022-11-26T00:33:49.826+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 90 for task http_check_copy (Bash command failed. The command returned a non-zero exit code 7.; 6042)
[2022-11-26T00:33:49.879+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2022-11-26T00:33:49.930+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
